crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -236,6 +236,9 @@ impl CairoHintProcessor<'_> {
                 print(inputs);
                 Ok(())
             }
+            "precalculate_address" => self
+                .cheated_state
+                .precalculate_address(&mut buffer, &inputs),
```

Comment:
The newest standard (from the https://github.com/foundry-rs/starknet-foundry/pull/419 which is going to be merged in 30 min) extracts inputs here and passes them as a separate values